test/unit/extending.js

```diff
@@ -71,7 +69,7 @@ describe('extending', () => {
     expect(el.constructor.extends).to.equal('div');
   });
 
-  if (canSetProto) {
+  if (Object.setPrototypeOf) {
```

Comment:
Same thing as your recently merged tests for static inheritance. I'll update to use that.